etc/kayobe/kolla/globals.yml

```diff
@@ -10,6 +10,10 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names }}{% endraw %}"
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
+# Use facts so this is determined correctly when the control host OS differs
+# from os_distribuition.
+kolla_base_distro_version: "{{ ansible_facts.distribution_major_version }}"
```

Comment:
It needs to be wrapped in `{% raw %}` like the one above to use the version of the remote host rather than the Ansible control host.